tests/ui/requirements.txt

```diff
@@ -7,3 +8,9 @@ pytest-selenium==1.11.1
 pytest-variables==1.7.0
 pytest-xdist==1.20.0
 selenium==3.5.0
+hashin==0.11.2
```

Comment:
Do we require hashin? Should we include the hashes for each of these packages?

Reply:
Imho, yes, we should start using hashes for the ui-test requirements file. Imho we could also move that file over to the `requirements` folder in the root folder to better reflect that it's there and make it easier to scan what dependencies we use for all our code.